src/backend/common/queries/database_query.py

```diff
@@ -46,3 +64,19 @@ def fetch_dict_async(self, version: ApiMajorVersion) -> TypedFuture[Dict]:
             safe_cast(QueryReturn, query_result)
         ).convert(version)
         return safe_cast(TypedFuture[Dict], res)
+
+    @classmethod
+    def _event_affected_queries(
```

Comment:
I don't love this approach but I also don't have any better ideas so I'm not going to push back